### PR TITLE
Update Welcome Page

### PR DESCRIPTION
**Pull request by TimCliff**
- Adds an introduction section at the top before the TOC
- Adds info for cover image
- Removes / updates out of date content
- Updates list of "Helpful Posts from Steemit Users"
- Removes Steemprentice (no longer supported)
- Adds Minnow Support Group

### CODE REVIEW

**Comment by originated** [comment]
Thanks @TimCliff good verbiage updates

**Comment by corkyk** [comment]
@TimCliff and @originated 

Gentlemen! There is an error in this pull request!

We are called the Minnow Support Project, not the Minnow Support Group!

Thank you!!!

@SirCork
Minnow Support Project

**Comment by TimCliff** [comment]
@corkyk - thanks so much for catching that! fixed.

**Comment by sneak** [comment]
@timcliff - we do not want to use the term "minnow", "whale", or "dolphin" anywhere in our official communications.

File: src/app/help/en/welcome.md [hunk 1]
```diff
@@ -356,13 +363,13 @@ It is not required either, but if you have other social media accounts (Twitter,
...
+Minnow Support Project is a group of people dedicated to helping new users find their way around Steemit. You can find them in the [Minnow Support Project channel](https://discord.gg/HYj4yvw) of Discord Chat.
 
 <a href="/welcome#Table_of_Contents">^</a>
 <a class="anchor" name="Third_Party_References"></a>
 ## Third Party References
 
-Peerhub, BlockTrades, Poloniex, Bittrex, Steemit Chat, Steemit Help, Steemprentice, as well as the tools listed under "Other Resources" are third party applications/services, and are not owned or maintained by Steemit, Inc. Their listing here does not constitute and endorsement or recommendation on behalf of Steemit, Inc.
+Peerhub, BlockTrades, Bittrex, Steemit Chat, Steemit Help, Minnow Support Project, and Discord Chat, as well as the tools listed under "Other Resources" are third party applications/services, and are not owned or maintained by Steemit, Inc. Their listing here does not constitute and endorsement or recommendation on behalf of Steemit, Inc.
```
Comment (sneak) [inline src/app/help/en/welcome.md]: no class-warfare slang

Comment (TimCliff) [reply src/app/help/en/welcome.md]: I'll discuss with the MSP group and see how they would like to proceed. Seems like the options are to either remove that part from the PR or have them come up with an alternate name.

**Comment by TimCliff** [comment]
@sneak - changes made. The group decided to call themselves "New Members Support Community"

**Comment by roadscape** [comment]
@TimCliff @sneak can we merge this now?

**Comment by TimCliff** [comment]
Good to go from my end. @sneak's call.

**Pull request merged**